otto_test.go

```diff
@@ -1803,6 +1803,35 @@ func TestOttoInterrupt(t *testing.T) {
 	}
 }
 
+func TestOtterStdoutWriter(t *testing.T) {
+	tests := []struct {
+		name   string
+		script string
+		expect string
+	}{
+		{
+			name:   "console.log",
+			script: "console.log('hello', 'world')",
+			expect: "hello world\n",
+		},
+		{
+			name:   "console.err",
+			script: "console.error('error', 'world')",
+			expect: "error world\n",
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			var buf = &bytes.Buffer{}
+			vm := New(WithStdoutWriter(buf))
+			_, err := vm.Run(tc.script)
+			require.NoError(t, err)
+			require.Equal(t, tc.expect, buf.String())
+		})
+	}
+}
```

Comment:
suggestion: can we use sub tests vs table driven as its more concise and allows developers to run specific tests directly from their editor e.g. VS Code.

Also the use of just a declared buffer vs instantiated is more idiomatic.

Not tested the below, so there might be typos, but you get the idea 😃 
```suggestion
func TestOtto_WithConsoleOutput(t *testing.T) {
	t.Run('console.log', func(t *testing.T) {
		var buf bytes.Buffer
		vm := New(WithConsoleOutput(&buf))
		_, err := vm.Run("console.log('hello', 'world')")
		require.NoError(t, err)
		require.Equal(t, "hello world\n", buf.String())
	})
	
	t.Run('console.err', func(t *testing.T) {
		var buf bytes.Buffer
		vm := New(WithConsoleOutput(&buf))
		_, err := vm.Run("console.error('world')")
		require.NoError(t, err)
		require.Equal(t, "error world\n", buf.String())
	})
}
```